app/boards/arduino_giga_r1_stm32h747xx_m7.overlay

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2025 Felipe Neves
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/gpio/gpio.h>
```

Comment:
Is the `zephyr/dt-bindings/gpio/gpio.h` header needed?

Reply:
You're right! It is not nedded after all.

Let me fix